covid_xprize/examples/predictors/lstm/xprize_predictor.py

```diff
@@ -213,7 +213,7 @@ def _convert_ratios_to_total_cases(self,
     def _smooth_case_list(case_list, window):
         return pd.Series(case_list).rolling(window).mean().to_numpy()
 
-    def train(self):
+    def train(self, num_epochs=NUM_TRIALS):
```

Comment:
Something has to be renamed here... epochs and trials are not the same thing. I think the code is really using the variable as a number of trials, so you should probably rename `num_epochs` into `num_trials`.